modules/oe_list_pages_open_vocabularies/src/Event/SearchApiFacetUpdateEvent.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\oe_list_pages_open_vocabularies\Event;
+
+use Drupal\facets\FacetInterface;
+use Symfony\Component\EventDispatcher\Event;
+
+/**
+ * Event for updating search api facet configuration.
```

Comment:
Event fired when a facet is created from an open vocabularies association.

Allows to alter the created facet.